Subject: .DS_Store is stripped out by SVN, don't expect it
Author: Vincent Bernat
Forwarded: no

Index: cookiecutter/tests/test_generate.py
===================================================================
--- cookiecutter.orig/tests/test_generate.py	2013-12-30 14:56:44.137629436 +0100
+++ cookiecutter/tests/test_generate.py	2014-04-27 12:35:49.502988652 +0200
@@ -119,7 +119,6 @@
             repo_dir='tests/test-generate-binaries'
         )
         self.assertTrue(os.path.isfile('inputbinary_files/logo.png'))
-        self.assertTrue(os.path.isfile('inputbinary_files/.DS_Store'))
         self.assertTrue(os.path.isfile('inputbinary_files/readme.txt'))
         self.assertTrue(
             os.path.isfile('inputbinary_files/some_font.otf')
@@ -128,9 +127,6 @@
             os.path.isfile('inputbinary_files/binary_files/logo.png')
         )
         self.assertTrue(
-            os.path.isfile('inputbinary_files/binary_files/.DS_Store')
-        )
-        self.assertTrue(
             os.path.isfile('inputbinary_files/binary_files/readme.txt')
         )
         self.assertTrue(
