Subject: .DS_Store is stripped out by SVN, don't expect it
Author: Vincent Bernat
Forwarded: no

Index: cookiecutter-1.0.0/tests/test_generate_files.py
===================================================================
--- cookiecutter-1.0.0.orig/tests/test_generate_files.py
+++ cookiecutter-1.0.0/tests/test_generate_files.py
@@ -107,11 +107,9 @@ def test_generate_files_binaries():
     )
 
     assert os.path.isfile('inputbinary_files/logo.png')
-    assert os.path.isfile('inputbinary_files/.DS_Store')
     assert os.path.isfile('inputbinary_files/readme.txt')
     assert os.path.isfile('inputbinary_files/some_font.otf')
     assert os.path.isfile('inputbinary_files/binary_files/logo.png')
-    assert os.path.isfile('inputbinary_files/binary_files/.DS_Store')
     assert os.path.isfile('inputbinary_files/binary_files/readme.txt')
     assert os.path.isfile('inputbinary_files/binary_files/some_font.otf')
     assert os.path.isfile(
